### PR TITLE
Chatty butcher

### PR DESCRIPTION
**Pull request by jyuu**
Implements solution from #42 for the following set of objects: lm, glmnet, elnet, cv.glmnet, multnet, randomForest

### CODE REVIEW

File: R/ui.R [hunk 1]
```diff
@@ -5,47 +5,66 @@
...
-  }
-  if(!class_added) {
-    class_name <- "butchered"
-    ui_oops("Could not append additional {ui_value(class_name)} class")
+  if (is.null(mem)) {
+    ui_oops("No memory released. Do not butcher.")
+  } else {
+    ui_done("Memory released: {ui_value(mem)}")
+    if (!is.null(disabled)) {
+      ui_oops("Disabled: {ui_code(disabled)}")
```
Comment (DavisVaughan) [inline R/ui.R]: Would it make sense to have some kind of verbage saying that _at least_ these things have been disabled, but there might be other things that don't work anymore that we aren't sure about?

File: R/axe-generics.R [hunk 2]
```diff
@@ -1,38 +1,47 @@
...
-  assess_object(old, x, working = c("predict"))
+
+  temp_file <- tempfile(fileext = ".txt")
+  sink(file = temp_file)
+  x <- axe_call(x, verbose = verbose, ...)
+  x <- axe_ctrl(x, verbose = verbose, ...)
+  x <- axe_data(x, verbose = verbose, ...)
+  x <- axe_env(x, verbose = verbose, ...)
+  x <- axe_fitted(x, verbose = verbose, ...)
+  sink()
```
Comment (DavisVaughan) [inline R/axe-generics.R]: I feel like we might be able to come up with a better way around this. What if `axe_call()` always returned `x` with an attribute `butcher_disabled` which was a vector of character strings of the disabled function names?

Then inside `butcher()` you would set `verbose = FALSE` for all of the individual axe functions, but in that final `if (verbose)` check below, you could just extract out the `attr(x, "butcher_disabled")` object and pass that along to `assess_object()`?

File: R/ui.R (hunk 1 above)
Comment (jyuu) [reply R/ui.R]: good point! ive been testing every function specific to each modeling package on the axed model object, but haven't tested functions outside of the package that might use the axed modeling object. will think of a concise way to change the error message and detail this issue in the documentation

File: R/axe-generics.R (hunk 2 above)
Comment (jyuu) [reply R/axe-generics.R]: Oh, I like that idea. Thus these functions are carried along with the butchered model object  

Will update to reflect!

**Pull request merged**